ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master.yaml

```diff
@@ -40,6 +40,17 @@ tests:
     go test -v ./...
   container:
     from: src
+- as: verify-dev-csv
+  commands: |
```

Comment:
We always suggest that you put this into a script in your own repo -- `make validate-latest-csv` or similar, to keep the complexity low here.

Reply:
This specific validation is checking to make sure no changes to the source tree occur when the `make gen-latest-csv` command is invoked. I don't want to predict exactly what changes to look for, instead I just want to know if a clean source tree results in changes after running the command.

I think wrapping this logic up into a make target could cause some confusion as it will fail if unless the source tree is completely clean.